src/tests/public_api/test_helpers/receiver.h

```diff
@@ -36,7 +36,8 @@ class Receiver : public core::Thread {
              size_t num_chans,
              size_t frame_size,
              unsigned flags)
-        : recv_(NULL)
+        : Thread("roc_receiver")
```

Comment:
Here in below: to avoid inventing names for each test, since these names in tests are not important anyway, maybe let's just use same name like "roc_test", "test_thread", or something like this?

I don't want to make name optional, so we have compiler error if non-test code forgets to assign a name. But on the other hand maintaining those names in tests feels unnecessary.